### DISCUSSION TITLE
Welcome to BirdNET-system Discussions!

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-system#5] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together .

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guiu23** [comment]
If you need some help with this, tell me, I wanted to do exactly the same for my Raspberry and I just found it. 

I'll tell you when I'll have it installed and working. See you!

**Reply by mcguirepr89** [reply]
Awesome! Thank you for the offer to help out and for your interest in general. I've added a great deal since I started the project and will need to rewrite the installation before asking for any help -- there's some cruft and it's a little spaghetti-ish right now which makes it difficult to read. Once I've made it a little more modular, I'll feel less silly receiving help. That said, I would never discourage anyone from helping with this project because I love it so much, so please give it a spin and try and push it to its limits and we'll make it better together :) -- If you run into any issues with the installation, please let me know! Good luck and thanks again.

**Reply by mcguirepr89** [reply]
Hello, again! 
I just wanted to check in to see if you've been able to get things working?
I also wanted to let you know that I've completed and merged the new installation process into the BirdNET-system-for-raspi4 branch. Please consider this an invitation to look through the code and offer suggestions, pose any questions, rewrite things for clarity -- whatever is up your alley :)

My best regards,
Patrick

**Comment by mcguirepr89** [comment]
Greetings, @nilspupils!
I am just checking in to see if your microphone has arrived and whether you've been able to give the BirdNET-system any testing?

My best regards!
https://github.com/kahst/BirdNET/issues/20#issuecomment-921732661
